playbook-website/app/javascript/components/GlobalPropsAndTokens/ExamplesPage/Examples/DisplayImages/index.ts

```diff
@@ -5,6 +5,11 @@ import inline from './inline.svg';
 import inlineBlock from './inline-block.svg';
 import inlineFlex from './inline-flex.svg';
 import none from './none.svg';
+import absolute from './absolute.svg';
+import fixed from './fixed.svg';
+import relative from './relative.svg';
+import staticSvg from './static.svg';
+import sticky from './sticky.svg';
```

Comment:
These Position images should live in their own folder 'PositionImages' like we have for DisplayImages and CursorImages currently, to keep structure consistent! We should not be adding these to DisplayImages